paper-toast.html

```diff
@@ -111,25 +131,34 @@
           },
         },
 
+        /**
+         * @property visible
+         * @deprecated
```

Comment:
Unfortunately this doesn't appear in the docs. I see the `visible` property, but not the description. Am I missing something?